### PR TITLE
pt-br: text editing

### PR DESCRIPTION
**Pull request by atarrisse**
Text revision for translation to Brazilian Portuguese, based on English translation.

### CODE REVIEW

File: _ptbr/about.md [hunk 1]
```diff
@@ -5,11 +5,10 @@ title: Sobre
...
 # Sobre o livro
 
-Nos anos 90, durante a Guerra da Bósnia, nosso pai ficou limitado à
-Sarajevo (Saraievo), enquanto minha mãe, meu irmão Kosta e eu conseguimos fugir
-para a Sérvia. Apesar de tudo, nosso pai nos escrevia divertidas cartas, cheias de desenhos e positividade.
+Nos anos 90, durante a Guerra da Bósnia, nosso pai ficou em Sarajevo (Saraievo), enquanto minha mãe, meu irmão Kosta e eu conseguimos fugir
+para a Sérvia. Apesar de tudo, nosso pai nos escrevia cartas - divertidas, positivas e cheias de desenhos.
 
-Cerca de dez anos depois, o livro foi publicado. Esse livro consiste em cartas selecionadas do meu pai e as respostas que meu irmão Kosta escrevia em um caderno.
+Cerca de dez anos depois, o livro foi publicado. Este consiste em cartas selecionadas do meu pai e as respostas que meu irmão Kosta escrevia em um caderno.
```
Comment (fmedinac) [inline _ptbr/about.md]:
```suggestion
Cerca de dez anos depois, o livro foi publicado, consistindo em cartas selecionadas do meu pai e as respostas que meu irmão Kosta escrevia em um caderno.
```
?

File: _ptbr/letter-three.md [hunk 2]
```diff
@@ -25,12 +26,12 @@ title: Terceira carta
   </div>
 
   <div class="LetterThree-block LetterThree-block--two">
-    Ainda não temos <img 
+    Aqui os <img
```
Comment (fmedinac) [inline _ptbr/letter-three.md:29]:
```suggestion
    Aqui, os <img 
```
?

File: _ptbr/about.md (hunk 1 above)
Comment (atarrisse) [reply _ptbr/about.md]: 👍

File: _ptbr/reply-four.md [hunk 3]
```diff
@@ -9,11 +9,11 @@ title: Quarta resposta
 
 Querido pai,
 
-Fiquei feliz com a sua carta, então a mamãe entrou no meu quarto chorando. Ela soube que o tio Grujo morreu. Fui com ela até as autoridades porque ela precisava de documentos para viajar de trem de graça. Demorou horas e eles quase não deram os documentos pra ela. Ela foi ao funeral esta manhã e me pediu para tomar conta do Stane. Ele está lá fora brincando com o Ivica. Estou os estou olhando brincar pela janela. Eu não contei para o Stane onde a mamãe foi. Eu só disse a ele que ela voltará amanhã.
+Fiquei feliz com a sua carta, então a mamãe entrou no meu quarto chorando. Ela soube que o tio Grujo morreu. Fui com ela até as autoridades, para ela pedir um documento que a permite viajar de trem de graça. Demorou horas e eles quase não deram os documentos pra ela. Ela foi ao funeral esta manhã e me pediu para tomar conta do Stane. Ele está lá fora brincando com o Ivica. Estou os estou observando brincar pela janela. Eu não contei para o Stane onde a mamãe foi. Eu só disse a ele que ela voltará amanhã.
```
Comment (fmedinac) [inline _ptbr/reply-four.md]:
```suggestion
Fiquei feliz com a sua carta, então a mamãe entrou no meu quarto chorando. Ela soube que o tio Grujo morreu. Fui com ela até as autoridades, para ela pedir um documento que a permite viajar de trem de graça. Demorou horas e eles quase não deram os documentos pra ela. Ela foi ao funeral esta manhã e me pediu para tomar conta do Stane. Ele está lá fora brincando com o Ivica. Os estou observando brincar pela janela. Eu não contei para o Stane onde a mamãe foi. Eu só disse a ele que ela voltará amanhã.
```

File: _ptbr/letter-three.md (hunk 2 above)
Comment (atarrisse) [reply _ptbr/letter-three.md:29]: Hm... I think such a comma adds a pause in this sentence that doesn't reflect the tone from the english text.

File: _ptbr/reply-one.md [hunk 4]
```diff
@@ -9,19 +9,19 @@ title: Primeira resposta
...
+Já faz seis meses desde que saímos de Sarajevo no ônibus sem janelas. Enquanto o ônbus partia eu lhe via acenar e lhe acompanhei com o olhar até se perder à distância. "É bom que o ônibus não tenha janelas", pensei, "caso contrário eu ficara enjoado". Passamos por soldados que ocasionalmente gritavam algo para a gente. Todos vestiam uniformes de cor diferente.
 
-Foi aí que o ônibus quebrou. Bem ao lado de um grupo de soldados. Eles ordenaram que descessemos e revistaram toda a bagagem. Um deles, com chapéu de cowboy e bigode, falava em inglês com outro soldado de boina azul. Ele notou que eu o observava, acariciou minha cabeça e disse-me: "Não se preocupe, tudo vai ficar bem"
+Aí o ônibus quebrou, bem ao lado de um grupo de soldados. Eles ordenaram que todos saíssem do ônbus e começaram a revistar toda a bagagem. Um deles, com chapéu de cowboy e bigode, falava em inglês com outro soldado de boina azul. Ele reparou que eu o observava, deu tapinhas minha cabeça e me disse: "Não se preocupe, tudo vai ficar bem".
```
Comment (fmedinac) [inline _ptbr/reply-one.md]:
```suggestion
Aí o ônibus quebrou, bem ao lado de um grupo de soldados. Eles ordenaram que todos saíssem do ônbus e começaram a revistar toda a bagagem. Um deles, com chapéu de cowboy e bigode, falava em inglês com outro soldado de boina azul. Ele reparou que eu o observava, deu tapinhas na minha cabeça e me disse: "Não se preocupe, tudo vai ficar bem".
```

File: _ptbr/reply-four.md (hunk 3 above)
Comment (atarrisse) [reply _ptbr/reply-four.md]: 👍

File: _ptbr/reply-one.md [hunk 5]
```diff
@@ -9,19 +9,19 @@ title: Primeira resposta
...
 
-Esperamos na multidão até um novo ônibus aparecer. Quando ele finalmente chegou, já estava lotado e tinha janelas. Eu sabia que estaria nele (e que ele me deixaria doente).
+Esperamos todos juntos pelo próximo ônibus. Quando ele finalmente chegou, já estava lotado e tinha janelas. Eu sabia que ia enjoar.
 
-Mamãe abriu caminho no meio da multidão e conseguiu nos levar para o ônibus. Ela nos disse que nos encontraria assim que pegasse nossas mochilas. Eu estava com Stane em um ônibus lotado tentando ver a porta. Quando o ônibus começou a andar, comecei a chorar, apertando os ombros de Stane. A próxima coisa que me lembro foi as pessoas ao meu redor me dando tapinhas nos ombros e dizendo: "Não se assuste, olha lá, sua mãe está aqui..."
+Mamãe abriu caminho no meio da multidão e conseguimos lugar o ônibus. Ela disse que se juntaria a nós assim que pegasse nossas mochilas. Eu fiquei com Stane no um ônibus superlotado, tentando ver a porta. Quando o ônibus começou a andar, comecei a chorar, apertando os ombros de Stane. A próxima coisa que me lembro são as pessoas ao meu redor sendo afetuosas e dizendo: "Não se preocupe, olha lá, sua mãe está aqui."
```
Comment (fmedinac) [inline _ptbr/reply-one.md]:
```suggestion
Mamãe abriu caminho no meio da multidão e conseguimos lugares no ônibus. Ela disse que se juntaria a nós assim que pegasse nossas mochilas. Eu fiquei com Stane no um ônibus superlotado, tentando ver a porta. Quando o ônibus começou a andar, comecei a chorar, apertando os ombros de Stane. A próxima coisa que me lembro são as pessoas ao meu redor sendo afetuosas e dizendo: "Não se preocupe, olha lá, sua mãe está aqui."
```

File: _ptbr/reply-one.md (hunk 4 above)
Comment (atarrisse) [reply _ptbr/reply-one.md]: 👍

File: _ptbr/reply-one.md [hunk 6]
```diff
@@ -9,19 +9,19 @@ title: Primeira resposta
...
 
 Eu vomitei naquele ônibus.
 
-Eu me lembro de você dizendo ao se despedir que se juntaria a nós em breve e que enquanto isso, a gente deveria aproveitar Belgrado. Mas não estamos em Belgrado. Vivemos em uma vilinha do interior chamada Divljana, perto de Niš, em um >>resort chamado "Sutjeska". Espero que você venha pra cá em breve.
+Eu me lembro de você dizendo que se juntaria a nós em breve e que enquanto isso, a gente deveria aproveitar Belgrado. Mas não estamos em Belgrado. Estamos em uma pequena vila do interior chamada Divljana, perto de Niš, em um >>resort chamado "Sutjeska". Espero que você venha pra cá em breve.
```
Comment (fmedinac) [inline _ptbr/reply-one.md]:
```suggestion
Eu me lembro de você dizendo que se juntaria a nós em breve e que enquanto isso, a gente deveria aproveitar Belgrado. Mas não estamos em Belgrado. Estamos em uma pequena vila do interior chamada Divljana, perto de Niš, em um condomínio chamado "Sutjeska". Espero que você venha pra cá em breve.
```

File: _ptbr/reply-one.md (hunk 5 above)
Comment (atarrisse) [reply _ptbr/reply-one.md]: 👍

File: _ptbr/reply-three.md [hunk 7]
```diff
@@ -11,11 +11,11 @@ Querido pai,
 
 Sua carta chegou enquanto a mamãe e eu estávamos no hospital. (A vovó Ruža do nosso quarto ficou de olho no Stane durante esse período.)
 
-Fomos ao hospital por comer cogumelos venenosos. A tia Hope, do primeiro andar, me levava até a floresta para colher cogumelos. Era interessante, ela falava sobre a natureza o tempo todo. À noite, voltávamos com cogumelos e todos da sala jantavam juntos. Depois de um jantar desses, eu saí com o Mihajlo. Estávamos vendo a lua pelos binóculos, como se fosse um telescópio. De repente, senti uma fraqueza. Voltei para o quarto e reclamei pra minha mãe que estava doente. Todos se viraram para mim: "Você também?"
+Fomos parar no hospital por comer cogumelos venenosos. Frequentemente a tia Nada, do primeiro andar, me levava para a floresta para colhermos cogumelos. Era interessante, ela falava sobre a natureza o tempo todo. À noite voltávamos com cogumelos e todos do quarto jantavam juntos. Depois de um jantar desses, eu saí com o Mihajlo. Estávamos vendo a lua pelos binóculos, como se fosse um telescópio. De repente, senti uma fraqueza. Voltei para o quarto e contei pra mãe. Todos se viraram para mim: "Você também?"
```
Comment (fmedinac) [inline _ptbr/reply-three.md]:
```suggestion
Fomos parar no hospital por comer cogumelos venenosos. Frequentemente a tia Nada, do primeiro andar, me levava até a floresta para colhermos cogumelos. Era interessante, ela falava sobre a natureza o tempo todo. À noite voltávamos com cogumelos e todos do quarto jantavam juntos. Depois de um jantar desses, eu saí com o Mihajlo. Estávamos vendo a lua pelos binóculos, como se fosse um telescópio. De repente, senti uma fraqueza. Voltei para o quarto e contei pra mãe. Todos se viraram para mim: "Você também?"
```

File: _ptbr/reply-two.md [hunk 8]
```diff
@@ -11,12 +11,12 @@ Querido pai,
...
+Aqui em Divljana, fiz um bom amigo, Mihajlo. (Samir ainda é meu melhor amigo, mesmo que eu não o tenha visto desde que saímos de Sarajevo). Mihajlo mora no térreo do "Sutjeska", nosso resort. Há alguns dias, encontraram uma cobra dentro do quarto dele. Eu acho que a cobra não vai subir até o nosso quarto, que fica no segundo andar.
 
-Assim que cheguei ao "Sutjeska", Mihajlo me explicou que estávamos em guerra com os meninos de outro resort chamado "Igman". Imediatamente atacamos alguns deles com pinhas. Como haviam muitos, recuamos e os observamos de longe através dos binóculos do Mihajlo. Ele me disse que se mudaria logo e deixaria os binóculos para mim.
+Assim que cheguei ao "Sutjeska", Mihajlo me explicou que estávamos em guerra com os meninos de outro resort, chamado "Igman". Imediatamente atacamos alguns deles com pinhas. Como eles estavam em maior número, recuamos e os observamos de longe através dos binóculos do Mihajlo. Ele me disse que se mudaria logo e que deixaria os binóculos para mim.
```
Comment (fmedinac) [inline _ptbr/reply-two.md]:
```suggestion
Assim que cheguei ao "Sutjeska", Mihajlo me explicou que estávamos em guerra com os meninos de outro condomínio, chamado "Igman". Imediatamente atacamos alguns deles com pinhas. Como eles estavam em maior número, recuamos e os observamos de longe através dos binóculos do Mihajlo. Ele me disse que se mudaria logo e que deixaria os binóculos para mim.
```

File: _ptbr/reply-two.md [hunk 9]
```diff
@@ -11,12 +11,12 @@ Querido pai,
...
 
-Tenho orgulho de pertencer ao "Sutjeska", embora os do "Igman" sejam mais numerosos. Além de Mihajlo e eu, só temos dois meninos mais novos no "Sutjeska" que são de pouca utilidade, e os mais velhos não participam da guerra. Mihajlo e eu somos os líderes de "Sutjeska", e o líder do "Igman" é o Siniša, também da quarta série.
+Tenho orgulho de pertencer ao "Sutjeska", embora hajam mas meninos no "Igman". Além de Mihajlo e eu, só temos dois meninos mais novos no "Sutjeska" que são de pouca utilidade, e os mais velhos não participam da guerra. Mihajlo e eu somos os líderes de "Sutjeska", e o líder do "Igman" é o Siniša, também da quarta série.
```
Comment (fmedinac) [inline _ptbr/reply-two.md]:
```suggestion
Tenho orgulho de pertencer ao "Sutjeska", embora hajam mais meninos no "Igman". Além de Mihajlo e eu, só temos dois meninos mais novos no "Sutjeska" que são de pouca utilidade, e os mais velhos não participam da guerra. Mihajlo e eu somos os líderes de "Sutjeska", e o líder do "Igman" é o Siniša, também da quarta série.
```

File: _ptbr/reply-two.md [hunk 10]
```diff
@@ -11,12 +11,12 @@ Querido pai,
...
 
-O Stane também fez uma bom amigo, o Ivica. Como ainda são muito pequenos para a escola, eles brincam o dia inteiro. Eles só não gostam do Bora, o gordo do restaurante. Ele força as crianças a mergulharem as mãos em um líquido fedido antes de comer. Depois, suas mãos ficam completamente rachadas. Bora grita com Stane e Ivica sempre que eles tentam evitar essa água fedorenta. Stane mal pode esperar para você vir e mostrar para o Gordo Bora quem é que manda aqui. Eu mal posso esperar você chegar.
+O Stane também fez um bom amigo, o Ivica. Como ainda são muito pequenos para a escola, eles brincam o dia inteiro. Eles só não gostam do Bora, o gordo do restaurante. Ele força as crianças a mergulharem as mãos em um líquido fedido. Depois, suas mãos ficam completamente rachadas. Bora grita com Stane e Ivica sempre que eles tentam evitar essa água fedorenta. Stane mal pode esperar para você vir e mostrar para o Gordo Bora quem é que manda aqui. Eu mal posso esperar você chegar também.
```
Comment (fmedinac) [inline _ptbr/reply-two.md]:
```suggestion
O Stane também fez um bom amigo, o Ivica. Como ainda são muito pequenos para a escola, eles brincam o dia inteiro. Eles só não gostam do Bora, o gordo do restaurante. Ele força as crianças a mergulharem as mãos em um líquido fedido antes de comer. Depois, suas mãos ficam completamente rachadas. Bora grita com Stane e Ivica sempre que eles tentam evitar essa água fedorenta. Stane mal pode esperar para você vir e mostrar para o Gordo Bora quem é que manda aqui. Eu mal posso esperar você chegar também.
```

File: _ptbr/reply-one.md (hunk 6 above)
Comment (atarrisse) [reply _ptbr/reply-one.md]: I'm not 100% sure on that translation, for we use the word _resort_ in pt-br as well. I usually see "condo" used in en-us to describe what we would call "condomínio". Let's double check with original text.

File: _ptbr/reply-three.md (hunk 7 above)
Comment (atarrisse) [reply _ptbr/reply-three.md]: I disagree on this one: "até" gives me a sense of "until", in which case they'd not enter the forest. I suggest we double check this one as well 🤓

File: _ptbr/reply-two.md (hunk 8 above)
Comment (atarrisse) [reply _ptbr/reply-two.md]: > I'm not 100% sure on that translation, for we use the word resort in pt-br as well. I usually see "condo" used in en-us to describe what we would call "condomínio". Let's double check with original text.

File: _ptbr/reply-two.md (hunk 9 above)
Comment (atarrisse) [reply _ptbr/reply-two.md]: 🤦‍♀ 👍

File: _ptbr/reply-two.md (hunk 10 above)
Comment (atarrisse) [reply _ptbr/reply-two.md]: 👍

File: _ptbr/reply-one.md (hunk 6 above)
Comment (fmedinac) [reply _ptbr/reply-one.md]: I discussed it with @Stanko and we couldn't find a good word for that. It's like a "conjunto habitacional" built for the working class by the USSR government. Resort sounds like a vacation place here in Brazil.

This is how it looks like:

Comment (atarrisse) [reply _ptbr/reply-one.md]: Ok, resort is not the word then! I'll try get some other insights and get back at it!

File: _ptbr/reply-three.md (hunk 7 above)
Comment (atarrisse) [reply _ptbr/reply-three.md]: Agreed on "na" 👍

**Comment by Stanko** [comment]
Just let me know when you guys are comfortable with merging this one 🙂 

Thank you for doing this 🙏

File: _ptbr/reply-one.md (hunk 6 above)
Comment (fmedinac) [reply _ptbr/reply-one.md]: @Stanko we will replace "resort" by "conjunto residencial", that's the most similar expression we have in pt-br. We will add a translation note in another PR so you can decide if you include it or not.

Comment (Stanko) [reply _ptbr/reply-one.md]: Let's keep it like that then 👍

File: _ptbr/reply-two.md (hunk 8 above)
Comment (atarrisse) [reply _ptbr/reply-two.md]: > we will replace "resort" by "conjunto residencial", that's the most similar expression we have in pt-br. We will add a translation note in another PR so you can decide if you include it or not.

**Review by fmedinac — APPROVED** [review]

**Review by fmedinac — APPROVED** [review]

**Pull request merged**